profiles/themes/Salad.css

```diff
@@ -0,0 +1,72 @@
+/*
+
+Theme: Salad
+Author: YukioKoito
+Version: 1.0
+Description: salad theme after the salad.com color scheme
```

Comment:
questionable brand placement

Reply:
this is their color scheme and I used it, reasonable to mention them there.

Reply:
L